### ISSUE TITLE
Investigation: compare local build of 1.8.0-1 with release

### ISSUE THREAD

**Issue by oubiwann** [clojusc/osgi#5]
I'm seeing a different behaviour between the local build of 1.8.0-2 and 1.9.0-3. There are several things that have changed, so this work will help remove variables.  If these both behave the same, I can safely assume I'm producing a local clojure.osgi bundle that behaves like the released bundle and that the local build process is not to blame for the difference in behaviour.

Part of #4.

**Comment by oubiwann** [comment]
A local build of clojure.osgi 1.8.0-1 (as 1.8.0-2-SNAPSHOT) does not exhibit the same behaviour as the actual 1.8.0-1 release. In particular, when starting the example bundle from here:
 * https://github.com/starship-tools/farana/tree/master/examples/example1
(with the appropriate configuration changes to point to the local install), startup looks like this:
```
$ ./bin/felix 

Starting Gogo, the Felix shell ...
To exit Gogo, type ^D

____________________________
Welcome to Apache Felix Gogo

g! Starting to listen for service events ...         
```
Whereas with the actual release, startup looks like this:

```
Starting Gogo, the Felix shell ...
To exit Gogo, type ^D

Starting to listen for service events ...
Service of type org.apache.felix.bundlerepository.RepositoryAdmin registered.
Service of type org.osgi.service.repository.Repository registered.
Service of type org.apache.felix.bundlerepository.impl.ObrGogoCommand registered.
Service of type org.osgi.service.url.URLStreamHandlerService registered.
Service of type org.apache.felix.gogo.command.Basic registered.
Service of type org.apache.felix.gogo.command.Inspect registered.
Service of type org.apache.felix.gogo.command.Files registered.
Service of type org.apache.felix.service.threadio.ThreadIO registered.
Service of type org.apache.felix.service.command.Converter registered.
Service of type org.apache.felix.gogo.jline.Builtin registered.
Service of type org.apache.felix.gogo.jline.Procedural registered.
Service of type org.apache.felix.gogo.jline.Posix registered.
Service of type org.apache.felix.gogo.jline.Shell registered.
Service of type org.apache.felix.service.command.CommandProcessor registered.
____________________________
Welcome to Apache Felix Gogo
```

**Comment by oubiwann** [comment]
My local installs have been with different artifact and group ids, in particular: `clojusc` and `osgi`, as opposed to `com.theoryinpractise` and `clojure.osgi`.

I tried doing a local build using the same names as upstream (`com.theoryinpractise/clojure.osgi`), and that _did_ work.

I also tried `clojusc/clojure.osgi`, and that did too. So ... the artifact id might need to be the same as the namespace?

The tests mentioned above (in this comment) were all done with 1.8.0-1 and -2; I will test with the latest ...
